### PR TITLE
fix(auth): revert session debouncer

### PR DESCRIPTION
**Pull request by AllanZhengYP**
#### Description of changes

The new user session debouncer [causes integration tests to fail](https://app.circleci.com/pipelines/github/aws-amplify/amplify-js/16568/workflows/a2463b41-b05a-4d53-9116-b4ec015c9c2f). The failed datastore auth integ tests is caused by race conditions between the integ test cases, likely because the signout is taking longer than before.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -503,7 +503,7 @@ jobs:
           framework: vue
           category: auth
           sample_name: authenticator-vue3
-          spec: new-ui-authenticator
```
Comment (stocaaro) [inline .circleci/config.yml]: Seems like we updated the app.  Maybe keep the changes from this file?

**Review by jimblanc — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -1314,6 +1314,7 @@ releasable_branches: &releasable_branches
       - geo/main
       - in-app-messaging/main
       - next-major-version/5
+      - revert-debouncer
```
Comment (stocaaro) [inline .circleci/config.yml:1317]: Drop this before we merge to next

**Review by stocaaro — COMMENTED** [review]
LGTM. Glad we're testing before this merged.

**Review by elorzafe — APPROVED** [review]
`Auth.ts` changes looks good to me

Thanks @AllanZhengYP

**Review by stocaaro — APPROVED** [review]

**Pull request merged**